ml-agents-envs/mlagents/envs/env_manager.py

```diff
@@ -0,0 +1,38 @@
+from abc import ABC, abstractmethod
+from typing import List, Dict, NamedTuple, Optional
+from mlagents.envs import AllBrainInfo, BrainParameters, Policy, ActionInfo
+
+
+class StepInfo(NamedTuple):
+    last_all_brain_info: Optional[AllBrainInfo]
```

Comment:
"previous" instead of "last"? I think that's more consistent with other variables in the repo.